adapters/signer_extraction_adapter/signer_extraction_adapter.go

```diff
@@ -0,0 +1,51 @@
+package signerextraction
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/signing"
+)
+
+type SignerData struct {
+	Signer   sdk.AccAddress
```

Comment:
Curious to hear what you think, but I wonder if this needs to be an `sdk.AccAddress` or just a string. Also, we should probably bring this up to the cosmos sdk team. Ideally, we upstream these changes for others to also have, but for time being this works.